### PR TITLE
PMM-2314 Instructing a user how to appropriately estimate disk space requireme…

### PR DESCRIPTION
**Pull request by fiowro**
…nts for PMM when deploying via AWS Marketplace

### CODE REVIEW

File: doc/source/deploy/server/ami.rst [hunk 1]
```diff
@@ -27,6 +27,15 @@ later, and this table serves the information purposes, to plan costs.
    see its price. |pmm| comes for no cost, you may only need to pay for the
    infrastructure provided by |amazon|.
 
+.. note::
+
+   Disk space used by |pmm-server| highly depends on the amount of hosts under
```
Comment (michaelcoburn) [inline doc/source/deploy/server/ami.rst]:
```suggestion
   Disk space consumed by |pmm-server| depends on the number of hosts under
```

File: doc/source/deploy/server/ami.rst [hunk 2]
```diff
@@ -27,6 +27,15 @@ later, and this table serves the information purposes, to plan costs.
...
+   monitoring. There is an approximate formula tested on the PMM Demo <https://pmmdemo.percona.com/>`_
```
Comment (michaelcoburn) [inline doc/source/deploy/server/ami.rst]:
```suggestion
   monitoring. Each environment will be unique, however consider modeling your data consumption based on PMM Demo <https://pmmdemo.percona.com/>`_
```

File: doc/source/deploy/server/ami.rst [hunk 3]
```diff
@@ -27,6 +27,15 @@ later, and this table serves the information purposes, to plan costs.
...
+   web site, which estimates allocated space as ~230MB/host/day, or ~6.9GB/host
```
Comment (michaelcoburn) [inline doc/source/deploy/server/ami.rst]:
```suggestion
   web site, which consumes ~230MB/host/day, or ~6.9GB/host
```

File: doc/source/deploy/server/ami.rst [hunk 4]
```diff
@@ -27,6 +27,15 @@ later, and this table serves the information purposes, to plan costs.
...
+   in case of 30 day retention period, which is set by default. See `this blog post <https://www.percona.com/blog/2017/05/04/how-much-disk-space-should-i-allocate-for-percona-monitoring-and-management/>`_ for more
```
Comment (michaelcoburn) [inline doc/source/deploy/server/ami.rst]:
```suggestion
at the default 30 day retention period. See `this blog post <https://www.percona.com/blog/2017/05/04/how-much-disk-space-should-i-allocate-for-percona-monitoring-and-management/>`_ for more
```

File: doc/source/deploy/server/ami.rst [hunk 5]
```diff
@@ -27,6 +27,15 @@ later, and this table serves the information purposes, to plan costs.
...
+   details about the |pmm-server| disk space consumption and its more precise
```
Comment (michaelcoburn) [inline doc/source/deploy/server/ami.rst]:
```suggestion
   details.
```

File: doc/source/deploy/server/ami.rst [hunk 6]
```diff
@@ -27,6 +27,15 @@ later, and this table serves the information purposes, to plan costs.
...
+   planning.
```
Comment (michaelcoburn) [inline doc/source/deploy/server/ami.rst]:
```suggestion

```

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-243/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-243/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-243/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-243/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-243/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-243/

**Pull request merged**